### PR TITLE
some of manage table optimization and reinforce configuration's validation

### PR DESCRIPTION
**Pull request by wenyh1**
Reason:  
  Improve inner 505、 512、514、515、516
Type:  
  improve  
Influences：  
  some of manage table optimization and reinforce configuration's validation

### CODE REVIEW

**Review by PanternBao — APPROVED** [review]
lgtm

**Comment by wenyh1** [comment]
please wait

**Comment by wenyh1** [comment]
please again review

**Review by PanternBao — APPROVED** [review]
lgtm

File: src/main/java/com/actiontech/dble/services/manager/information/tables/DbleEntryTablePrivilege.java [hunk 1]
```diff
@@ -56,77 +58,75 @@ protected void initColumnAndType() {
...
+                            boolean noEffective = (shardingUserConfig.isReadOnly() || !userPrivilegesConfig.isCheck());
+                            Map<String, UserPrivilegesConfig.SchemaPrivilege> schemaPrivilege = userPrivilegesConfig.getSchemaPrivileges();
+                            if (!CollectionUtil.isEmpty(schemaPrivilege)) {
+                                schemaPrivilege.forEach((schema, sPrivilege) -> {
+                                    if (shardingUserConfig.getSchemas().contains(schema)) {
+                                        SchemaConfig schemaConfig = DbleServer.getInstance().getConfig().getSchemas().get(schema);
+                                        if (schemaConfig != null) {
+                                            Set<String> tables = new HashSet<>(schemaConfig.getTables().keySet());
+                                            Set<String> metas = new HashSet<>(ProxyMeta.getInstance().getTmManager().getCatalogs().get(schema).getTableMetas().keySet());
+                                            Set<String> viewMetas = ProxyMeta.getInstance().getTmManager().getCatalogs().get(schema).getViewMetas().keySet();
```
Comment (PanternBao) [inline src/main/java/com/actiontech/dble/services/manager/information/tables/DbleEntryTablePrivilege.java:87]: I can't understand this

**Review by PanternBao — APPROVED** [review]
lgtm

**Pull request merged**